### PR TITLE
docs: Add README.md to packages

### LINKED ISSUE #134: Create README.md pages for packages
Now we have only root `README.md` so published on npm packages from this monorepo does not have any description. Some of the users of our packages want these packages to be documented. For start, it would be enough to create `packages/*/README.md` files with a brief explanation of the purposes of each package.

### PR DESCRIPTION
**Pull request by sutarmin**
Fixes #134 

This is a way to start supporting README-files, not an exhaustive description.
Comments about typos and other mistakes are appreciated.

### CODE REVIEW

File: packages/lint/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[![build status](https://img.shields.io/travis/devex-web-frontend/dx-platform/master.svg?style=flat-square)](https://travis-ci.org/devex-web-frontend/dx-platform) ![npm](https://img.shields.io/npm/v/@devexperts/lint.svg?style=flat-square) 
+## @devexperts/lint
+Infrastucture confuguration files, to have consistent linting rules across different Devexperts frontend team projects. Those files are:
```
Comment (evgeniy-ugryumov) [inline packages/lint/README.md]:
```suggestion
Infrastructure configuration files, to have consistent linting rules across different Devexperts frontend team projects. Those files are:
```

File: packages/tools/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+```
+dx-tools <command>
+```
+Commands are located under `src/scripts` directory. Some of available commands:
+#### build-lib
+```
+dx-tools build-lib <src-path> <dist-path> [-w|--watch] [-f|--failOnError]
+```
+Builds projects using following rules:
+* __Typescript__ - transpile `*.ts|*.tsx` files, produce `*.js|*.jsx` & `*.d.ts` files, run typechecking service in background
```
Comment (evgeniy-ugryumov) [inline packages/tools/README.md]:
```suggestion
* __Typescript__ - transpile `*.ts|*.tsx` files, produce `*.js|*.jsx` & `*.d.ts` files, run type checking service in background
```

File: packages/utils/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[![build status](https://img.shields.io/travis/devex-web-frontend/dx-platform/master.svg?style=flat-square)](https://travis-ci.org/devex-web-frontend/dx-platform) ![npm](https://img.shields.io/npm/v/@devexperts/utils.svg?style=flat-square) 
+## @devexperts/utils
+Commonly used across the different projects utilities.
```
Comment (evgeniy-ugryumov) [inline packages/utils/README.md]:
```suggestion
Commonly used across different projects utilities.
```

**Comment by sutarmin** [comment]
Thanks, @evgeniy-ugryumov, applied your suggestions :) @scink, as one of the most demanding to everything team members, could you take a look at this one?

**Pull request merged**